src/alexapi/device_platforms/dummy.py

```diff
@@ -11,36 +14,28 @@ def __init__(self, config):
 		self.should_confirm_trigger = False
 
 	def setup(self):
-		if self._pconfig['verbose']:
```

Comment:
I think that it makes sense to make "logging" option in config file, so anybody can set it permanently to DEBUG, if they want to. Default will be INFO and `-d` option for `main.py` will override it at runtime.